### PR TITLE
common/defs: add Particl & testnet support

### PR DESCRIPTION
**Pull request by kewde**

### CODE REVIEW

**Comment by prusnak** [comment]
Remove changes to common/defs/coins_details.json  in the commit. That file is autogenerated.

**Comment by kewde** [comment]
Thanks, fixed now!

**Comment by prusnak** [comment]
Please make the icon for testnet grayscale

**Comment by kewde** [comment]
Adjusted

File: common/defs/bitcoin/particl.json [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  "force_bip143": false,
+  "bip115": false,
+  "default_fee_b": {
+    "Low": 10,
+    "Economy": 70,
+    "Normal": 140,
+    "High": 200
+  },
+  "dust_limit": 546,
+  "blocktime_seconds": 600,
```
Comment (jsolman) [inline common/defs/bitcoin/particl.json]: Shouldn’t this be 120 for Particl?

Comment (PARTimprovert) [reply common/defs/bitcoin/particl.json]: correct

Comment (kewde) [reply common/defs/bitcoin/particl.json]: My bad, I looked over it. It's not relevant for our particl-core integration though.
I will submit a patch once we add support to the official trezor wallet, which will require modifying this file again to add the blockbook and insight URLs.

I have not experienced any issues with transaction or message signing so far.

**Pull request merged**